app/controllers/orders_controller.rb

```diff
@@ -336,7 +336,7 @@ def load_statuses
   end
 
   def order_params
-    return if params[:order].blank?
+    return {} if params[:order].blank?
```

Comment:
Should this be `ActionController::Parameters.new` or something rather than a hash for consistency.